oak_functions/examples/weather_lookup/README.md

```diff
@@ -57,6 +57,42 @@ radius of 40km. See https://s2geometry.io/devguide/examples/coverings for more
 information on coverings. Cell coverings for areas can be visualised using
 https://s2.sidewalklabs.com/regioncoverer/.
 
+### File format
+
+The weather file format looks as follows:
+
+Weather file is a Protobuf compiled file that consists of a list of the
+following entries:
+
+```protobuf
+message Entry {
+  bytes key = 1;
+  bytes value = 2;
+}
+```
+
+There are 2 types of entries:
+
+- Location entries
+- Cell entries
+
+_Location_ entries are represented as:
+
+- `key` is `LATITUDE_SIGNED_BIG_ENDIAN | LONGITUDE_SIGNED_BIG_ENDIAN`
+  [Length-Delimited](https://developers.google.com/protocol-buffers/docs/encoding#structure)
```

Comment:
It is probably not required to specify that this field is length-delimited. This is implied by the proto definition and handled internally by the protobuf encoding. If you want to specify it, you should also specify it for the value, because that is also length-delimited.